charts/optimize-live/values.yaml

```diff
@@ -4,41 +4,43 @@
 
 replicaCount: 1
 
-component:
+tsdb:
   image:
-    controller:
-      repository: registry.stormforge.io/optimize-live/controller
-      pullPolicy: IfNotPresent
-      tag: 0.0.19
-    tsdb:
-      repository: registry.stormforge.io/optimize-live/tsdb
-      pullPolicy: IfNotPresent
-      tag: 0.0.12
-    applier:
-      repository: registry.stormforge.io/optimize-live/applier
-      pullPolicy: IfNotPresent
-      tag: 0.0.6
-    recommender:
-      repository: registry.stormforge.io/optimize-live/recommender
-      pullPolicy: IfNotPresent
-      tag: 0.3.2
-    grafana:
-      repository: grafana/grafana-oss
-      pullPolicy: IfNotPresent
-      tag: 9.1.5
+    repository: registry.stormforge.io/optimize-live/tsdb
+    pullPolicy: IfNotPresent
+    tag: 0.0.12
+
+applier:
+  image:
+    repository: registry.stormforge.io/optimize-live/applier
+    pullPolicy: IfNotPresent
+    tag: 0.0.6
+
+recommender:
+  image:
+    repository: registry.stormforge.io/optimize-live/recommender
+    pullPolicy: IfNotPresent
+    tag: 0.3.2
   resources:
-    tsdb: {}
```

Comment:
note: ensure we pick up any component resources changes when dealing with any future merge conflicts.